packages/eslint-config-airbnb-base/rules/errors.js

```diff
@@ -58,10 +60,12 @@ module.exports = {
     // disallow assigning to the exception in a catch block
     'no-ex-assign': 'error',
 
+    // -- SITUATION ANALYSIS NEEDED. I'm torn on this. Though technically unnecessary, some may prefer the explicit boolean cast.
     // disallow double-negation boolean casts in a boolean context
     // https://eslint.org/docs/rules/no-extra-boolean-cast
     'no-extra-boolean-cast': 'error',
```

Comment:
Flip this to `off` due to current practices and the unlikeliness that we shift into this standard where we'd want it to `error` for the confusion that a casting can create.